scripts/sql/views/core_protocol_fees.sql

```diff
@@ -0,0 +1,50 @@
+CREATE VIEW core_protocol_fees as 
```

Comment:
what is this and why do i see this in 2 different PRs?

Reply:
This and the other one are the views that I'm using for the data warehouse. I've been re-running this sql to recreate the views when I switch over the data warehouse subgraph and I needed somewhere to store the sql query.

Eventually I want to just create the views automatically in each subgraph by running the sql query when the subgraph is created (like with the database triggers)